roadmap/implementors-guide/src/types/backing.md

```diff
@@ -0,0 +1,75 @@
+# Backing Types
+
+[Candidates](candidate.html) go through many phases before being considered included in a fork of the relay chain and eventually accepted.
+
+These types describe the data used in the backing phase. Some are sent over the wire within subsystems, and some are simply included in the relay-chain block.
+
+## Validity Attestation
+
+An attestation of validity for a candidate, used as part of a backing. Both the `Seconded` and `Valid` statements are considered attestations of validity. This structure is only useful where the candidate referenced is apparent.
+
+```rust
+enum ValidityAttestation {
+  /// Implicit validity attestation by issuing.
+  /// This corresponds to issuance of a `Seconded` statement.
+  Implicit(ValidatorSignature),
+  /// An explicit attestation. This corresponds to issuance of a
+  /// `Valid` statement.
+  Explicit(ValidatorSignature),
+}
+```
+
+## Statement Type
+
+The [Candidate Backing subsystem](/node/utility/candidate-backing.html) issues and signes these after candidate validation.
+
+```rust
+/// A statement about the validity of a parachain candidate.
+enum Statement {
+  /// A statement about a new candidate being seconded by a validator. This is an implicit validity vote.
+  ///
+  /// The main semantic difference between `Seconded` and `Valid` comes from the fact that every validator may
+  /// second only 1 candidate; this places an upper bound on the total number of candidates whose validity
+  /// needs to be checked. A validator who seconds more than 1 parachain candidate per relay head is subject
+  /// to slashing.
+  Seconded(CandidateReceipt),
+  /// A statement about the validity of a candidate, based on candidate's hash.
+  Valid(Hash),
+  /// A statement about the invalidity of a candidate.
+  Invalid(Hash),
+}
+```
+
+## Signed Statement Type
+
+The actual signed payload should reference only the hash of the CandidateReceipt, even in the `Seconded` case and should include
```

Comment:
It would be helpful to add a brief note of what precisely gets concatenated into the signed payload, in what order.